applications/mne_analyze/plugins/controlmanager/controlmanager.cpp

```diff
@@ -176,10 +176,12 @@ void ControlManager::handleEvent(QSharedPointer<Event> e)
 {
     switch (e->getType()) {
     case SELECTED_MODEL_CHANGED:
-        onScalingChanged(m_ScalingParameters.m_mScalingMap);
-        m_ViewParmeters.m_sViewsToApply = m_pApplyToView->getSelectedViews();
-        m_ViewParmeters.m_sSettingsToApply = ANSHAREDLIB::ViewParameters::ViewSetting::all;
-        m_pCommu->publishEvent(EVENT_TYPE::VIEW_SETTINGS_CHANGED, QVariant::fromValue(m_ViewParmeters));
+        if(e->getData().value<QSharedPointer<ANSHAREDLIB::AbstractModel> >()->getType() == ANSHAREDLIB_FIFFRAW_MODEL) {
```

Comment:
this will stop the scaling from working with the average model/view. maybe use `!= ANSHAREDLIB_BEMDATA_MODEL` instead.